wallet/api/openrpc.json

```diff
@@ -5,6 +5,82 @@
     "title": "Wallet API"
   },
   "methods": [
+    {
+      "name": "client.check_transaction",
+      "summary": "Check a transaction without sending it.",
```

Comment:
It does send the transaction to the network, but the network doesn't add it to the block. Maybe we can be clearer on htat part.

Reply:
Addressed here - https://github.com/vegaprotocol/vega/pull/7657/commits/061417ef72aebed33d0cbbc49c5b46f32310fe14#diff-4541166dc7cdf695bbc57461645d25da8509eeb5839947816d4979a654eeb880R10-R11